_data/toc.yaml

```
@@ -953,14 +953,12 @@ reference:
     path: /engine/reference/commandline/dockerd/
 - sectiontitle: Application Programming Interfaces (APIs)
   section:
-  - sectiontitle: Docker Engine API
+  - sectiontitle: Docker Engine SDK and API
```

Bit on the fence on this one; API is more likely something people would look for, so _if_ we want to include "sdk", I'd put it last, but perhaps even "keep it as-is" (see below)

```suggestion
  - sectiontitle: Docker Engine API and SDK
```

---

Added SDK first to be consistent with the topic title:
- https://docs.docker.com/develop/sdk/
- https://docs.docker.com/develop/sdk/examples/